Sources/XcbeautifyLib/Pattern.swift

```diff
@@ -314,12 +314,12 @@ enum Pattern: String {
     /// $1 = file path
     /// $2 = filename
     /// $3 = reason
-    case compileWarning = #"^(([^:]*):*\d*:*\d*):\swarning:\s(.*)$"#
+    case compileWarning = #"^(?!ld)(([^:]*):*\d*:*\d*):\swarning:\s(.*)$"#
 
     /// Regular expression captured groups:
     /// $1 = ld prefix
     /// $2 = warning message
-    case ldWarning = #"^(ld: )warning: (.*)"#
+    case ldWarning = #"^(ld: )warning: (?!.*duplicate symbol)(.*)"#
```

Comment:
Do you know if we can avoid the catchall (`.*`) here and on the modifications below? I recognize this is used by many of the other regexes, and there may not be a reliable alternative. That said, this has introduced some performance issues in the past, and it'd be helpful to remove them while we're here, if possible.

Reply:
I think it should become another issue.

`ld` related messages are become unstable in recent Xcode. We can enhance the performance later.